recipes/heavydb/meta.yaml

```
@@ -0,0 +1,562 @@
+{% set version = "6.0.0" %}  # PEP 386
+{% set source_version = version %}
+{% set base_version = version %}
+{% set number = "0" %}
+{% set cuda_enabled = cuda_compiler_version is not undefined and cuda_compiler_version == '11.0' %}
+{% set build_ext_version = "1.0.0" %}
+{% set build_ext = "cuda" if cuda_enabled else "cpu" %}
+{% set build_string = "h{}_{}".format(PKG_HASH, number) %}
+{% set build_string_ext = "{}_{}".format(build_string, build_ext) %}
+{% set py_build_string_ext = "py{}{}_{}".format(CONDA_PY, build_string, build_ext) %}
+{% set py_build_string = "py{}{}".format(CONDA_PY, build_string) %}
+{% set install_base = "opt/heavy" %}
+{% set arrow_version = "7.*" %}
+{% set arrow_proc_version = "3.*" %}
+{% set pythrift_version = "0.16.*" %}
+{% set thrift_version = "0.16.*" %}
+# heavydb 6.0.0 is not LLVM 12 ready
+{% set llvm_version = "11" %}
+
+package:
+  name: heavydb-ext
+  version: {{ version }}
+
+source:
+  url: https://github.com/heavyai/heavydb/archive/v{{ source_version }}.tar.gz
```

A temporary solution until 6.0.0 is released:
```suggestion
  # url: https://github.com/heavyai/heavydb/archive/v{{ source_version }}.tar.gz
  url: https://github.com/heavyai/heavydb/archive/master.tar.gz
```

---

I actually forgot this commit 😅